### DISCUSSION TITLE
申请退款，无法使用

### DISCUSSION THREAD

**Discussion by pplive98** [getlantern/lantern-forum-cn#992] [General]
10月23日购买了专业版，但是打不开网站，根本无法使用，联系客服也无人回应，申请退款！账户274892121@qq.com,支付截图如下
![微信图片_20241023190357](https://github.com/user-attachments/assets/cca47179-d208-4077-908c-1723da5a4d5a)
![微信图片_20241023190357](https://github.com/user-attachments/assets/60ace0a0-af42-47a8-ade8-194bd5c3987e)

**Comment by MaxMKMax** [comment]
@pplive98 您好. 我帮您提交上去. 谢谢!

**Comment by pplive98** [comment]
***@***.***，单据见图片。实在用不了，请帮忙退款-------- 原始邮件 --------发件人： Mike K ***@***.***>日期： 2024年10月26日周六 23:01收件人： getlantern/lantern-forum-cn ***@***.***>抄送： pplive98 ***@***.***>, Mention ***@***.***>主    题： Re: [getlantern/lantern-forum-cn] 申请退款，无法使用 (Discussion #992)
@pplive98 您好. 请您提供我支付单据和邮箱. 我帮您提交上去. 谢谢!
